app/build.gradle

```diff
@@ -48,12 +48,6 @@ android {
 
         ext.betaDistributionReleaseNotes = System.getenv("APP_RELEASE_NOTES")
 
-        javaCompileOptions {
```

Comment:
this will be added later eventually. Can we just leave it here?

Reply:
Shouldn't that be on the data module and not here, or is it required also here?

Reply:
I moved it to the `data` module because that's where we have the room entities. If you leave it in `app/build.gradle` it won't work.